bin/pycbc_multi_inspiral

```diff
@@ -661,9 +665,11 @@ with ctx:
                         bank_chisq.values(template, stilde[ifo].psd, stilde[ifo],
                                          coherent_ifo_triggers[ifo] /
                                          norm_dict[ifo], norm_dict[ifo],
-                                         coinc_idx+stilde[ifo].analyze.start)
+                                         coinc_idx_det_frame[ifo]+stilde[ifo].analyze.start)
+                    # snr_ts[ifo] is used here because autochisq runs into index error with snr[ifo] padded
+                    # by segment padding options
                     ifo_out_vals['cont_chisq'] = autochisq.values(
-                        snr[ifo], coinc_idx+stilde[ifo].analyze.start,
```

Comment:
Can you remind me what the different is between `snr` and `snr_ts`.

Reply:
`snr_ts` is the SNR dictionary directly from matched filtering. `snr` is the SNR dictionary that is normalized, affected by segment padding (i.e. the `--segment-start-pad` and `--segment-end-pad` options), and previously shifted to account for time delay. When segment padding options were used, the program would crash because `snr[ifo]` was shorter than the indices in `coinc_idx+stilde[ifo].analyze.start`. I was using `snr_ts` to account for this. After playing around with a debugger, it looks I can use `snr` and `coinc_idx_det_frame` without the `stilde[ifo].analyze.start` and get the same effect, which would probably be the better choice.